src/steps/account/account-field-equals.ts

```diff
@@ -82,6 +121,24 @@ export class AccountFieldEquals extends BaseStep implements StepInterface {
       return this.error('There was an error during validation of account field: %s', [e.message]);
     }
   }
+
```

Comment:
- [ ] Minor note on this section below: recommend limiting use of `titleCase` to just table headers.  Best to keep the key/value record keys (fields) matching the actual field values for possible copy/paste from error message to scenario definition.